website_odoo_debranding/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Tecnativa, S.L. - Vicent Cubells
+# Copyright 2018 Dreambits Technologies Pvt. Ltd., - Karan Shah
```

Comment:
usually we don't change copyright w/ significant contrib to the code. 
You could add yourself to the list of contributors.

Reply:
WIll fix it :)